generate_config.py

```diff
@@ -173,6 +174,9 @@
 /etc/nginx/ssl/""" + hostname + """.crt; # Wildcard SSL certificate
 /etc/nginx/ssl/""" + hostname + """.key; # Wildcard SSL key
 
+Note: by default self-registration is enabled. You can disable this by changing the 'self_registration' option in the config file to 
```

Comment:
Question about this, wouldn't setting this to a `no` still make it enabled? `no` is a string which should be True-ish in Python right: `if not settings['self_registration']:`. Let me know if I'm missing something!